tests/test_nodes.py

```diff
@@ -89,6 +90,59 @@ def test_insert_issue_in_a_more_complex_situation():
     assert str(document) == "<root><div1><div2/> </div1></root>"
 
 
+def test_an_issue():
```

Comment:
is there a way to make it more clear what the particular issue under test is and how this test is supposed to cover it?

Reply:
oops, that's a leftover from investigating the initial phenomenom. i'll add a comment.